rfcs/features/0041-renderer-plugin/README.md

```diff
@@ -0,0 +1,209 @@
+
+Let's take a look at the expanded document. 
+
+```json
+[
+  {
+    "https://www.w3.org/2018/credentials#credentialSubject": [
+      {
+        "https://example.org/examples#degree": [
+          {
+            "http://schema.org/name": [
+              {
+                "@type": "http://www.w3.org/1999/02/22-rdf-syntax-ns#HTML",
+                "@value": "Bachelor of Science and Arts"
+              }
+            ],
+            "@type": [
+              "https://example.org/examples#BachelorDegree"
+            ]
+          }
+        ],
+        "@id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+        "http://schema.org/name": [
+          {
+            "@type": "http://www.w3.org/1999/02/22-rdf-syntax-ns#HTML",
+            "@value": "Jane Smith"
+          }
+        ]
+      }
+    ],
+    "@id": "http://example.edu/credentials/3732",
+    "https://www.w3.org/2018/credentials#issuanceDate": [
+      {
+        "@type": "http://www.w3.org/2001/XMLSchema#dateTime",
+        "@value": "2010-01-01T00:00:00Z"
+      }
+    ],
+    "https://www.w3.org/2018/credentials#issuer": [
+      {
+        "@id": "https://example.edu/issuers/565049"
+      }
+    ],
+    "https://www.w3.org/2018/credentials#renderMethod": [
+      {
+        "@id": "<div style=\"width:300px; height:100px; border: 2px solid black; text-align:center\">\n  <div>\n    This {{credentialSubject.degree.name}} is conferred to\n  </div>\n  <strong style=\"font-size: 16px\">\n    {{credentialSubject.name}}\n  </strong>\n  <div>\n    by {{credentialSubject.degree.institution}}.\n  </div>\n</div>",
+        "@type": [
+          "WebRenderingTemplate2022"
+        ]
+      }
+    ],
+    "@type": [
+      "https://www.w3.org/2018/credentials#VerifiableCredential",
+      "https://example.org/examples#UniversityDegreeCredential"
+    ]
+  }
+]
+```
+
+We can see that the property named "render" in the original document is now expanded as the absolute URI for the renderMethod reserved term definition: https://www.w3.org/2018/credentials#renderMethod
+
+This is the value that the renderer-plugin inspects to locate the render method type. 
+
+> Note: the renderMethod is an array, but for the moment, the renderer-plugin assumes a single render method and only returns the rendering given by the first object in the array. Further work is required to develop an appraoch to handling multiple rendering options for a single document, this is likely to to include passing in a content type to the render-credential plugin, or an accepts header when calling remote server functions. 
+
+The vckit-core package contains a function for each known renderMethod type. Currently these are:
+- WebRenderingTemplate2022
+- SvgRenderingHint2022
+
+These functions are passed in to the renderer-plugin via a mapping defined in the agent configuration file. This allows for the addition of new handlers to the renderer-plugin via configuration, without changing the plugin itself. 
```

Comment:
As I understand, when implementing the renderer-plugin, we need to add the plugin and functions to the agent configuration file. But not sure when adding new handlers to the renderer-plugin, how will not change the plugin?